src/ci/scripts/install-clang.sh

```diff
@@ -12,17 +12,25 @@ source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
 LLVM_VERSION="10.0.0"
 
 if isMacOS; then
-    curl -f "${MIRRORS_BASE}/clang%2Bllvm-${LLVM_VERSION}-x86_64-apple-darwin.tar.xz" | tar xJf -
+    if [[ -s ${SELECT_XCODE} ]]; then
+        bin="${SELECT_XCODE}/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin"
 
-    ciCommandSetEnv CC "$(pwd)/clang+llvm-${LLVM_VERSION}-x86_64-apple-darwin/bin/clang"
-    ciCommandSetEnv CXX "$(pwd)/clang+llvm-${LLVM_VERSION}-x86_64-apple-darwin/bin/clang++"
+        ciCommandSetEnv CC "${bin}/clang"
+        ciCommandSetEnv CXX "${bin}/clang++"
```

Comment:
Is this intended to be temporary until we bump the LLVM version used in CI to one supporting macOS 11?

Reply:
I don't know, actually! That seems like it would be nice.

Reply:
I think it's really useful to converge macOS AArch64 to use the same upstream version as the other platforms, as it reduces the chance of encountering Apple Silicon-only bugs. I totally understand that's not the case today, but do you know if we expect it to be possible to use the next upstream version of LLVM instead of the one bundled with Xcode?

It'd be nice to have a comment on this in the source too!